### ISSUE TITLE
[Bug]: Static build for v1.35.1 installs netdata v1.35.0-2-gfee539f8e

### ISSUE THREAD

**Issue by matrey** [netdata/netdata#13222]
### Bug description

Static build for v1.35.1 installs netdata v1.35.0-2-gfee539f8e

```
$ bash netdata-x86_64-v1.35.1.gz.run --lsm
Begin3
Title:          netdata
Version:        v1.35.0-2-gfee539f8e
Description:    netdata is a system for distributed real-time performance and health monitoring.
                It provides unparalleled insights, in real-time, of everything happening on the
                system it runs (including applications such as web and database servers), using
                modern interactive web dashboards. netdata is fast and efficient, designed to
                permanently run on all systems (physical & virtual servers, containers, IoT
                devices), without disrupting their core function.
Keywords:       real-time performance and health monitoring
Author:         Costa Tsaousis (costa@tsaousis.gr)
Maintained-by:  Costa Tsaousis (costa@tsaousis.gr)
Original-site:  https://my-netdata.io/
Platform:       Unix
Copying-policy: GPL
End

```

The UI then mentions an update is available
![image](https://user-images.githubusercontent.com/5174727/175853281-6b00e8b3-7dcf-49aa-b536-48af903fa748.png)

### Expected behavior

Get the netdata version advertised in the static build name

### Steps to reproduce

Download static package from https://github.com/netdata/netdata/releases/download/v1.35.1/netdata-x86_64-v1.35.1.gz.run
Then run `bash netdata-x86_64-v1.35.1.gz.run --lsm`

### Installation method

other

### System info

```shell
Linux iZuf61p6r5j18beuvmrr8hZ 5.4.0-110-generic #124-Ubuntu SMP Thu Apr 14 19:46:19 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=20.04
/etc/lsb-release:DISTRIB_CODENAME=focal
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="20.04.4 LTS (Focal Fossa)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 20.04.4 LTS"
/etc/os-release:VERSION_ID="20.04"
/etc/os-release:VERSION_CODENAME=focal
/etc/os-release:UBUNTU_CODENAME=focal
```

### Netdata build info

```shell
Version: netdata v1.35.0-2-gfee539f8e
Configure options:  '--prefix=/opt/netdata/usr' '--sysconfdir=/opt/netdata/etc' '--localstatedir=/opt/netdata/var' '--libexecdir=/opt/netdata/usr/libexec' '--libdir=/opt/netdata/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--enable-cloud' '--without-bundled-protobuf' '--disable-dependency-tracking' '--with-bundled-libJudy' 'CFLAGS=-static -O2 -I/openssl-static/include -pipe' 'LDFLAGS=-static -L/openssl-static/lib' 'PKG_CONFIG_PATH=/openssl-static/lib/pkgconfig'
Install type: manual-static
    Binary architecture: x86_64
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    YES
    IPMI:                    NO
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
```

**Comment by MrZammler** [comment]
Hi guys, thanks for the reports, we will have a look asap.

**Comment by maneamarius** [comment]
For what is worth, I've tested the previous release `1.35.0` and it has a similar issue.
The version reported there is `v1.34.0-293-ge61633a9e`

**Comment by Ferroin** [comment]
More specifically, it’s something that’s _not_ in the `v1.34` branch, otherwise 1.34.1 would be affected as well. Unfortunately, this is not something we can easily bisect, and there’s a few hundred commits to go through...

**Comment by Ferroin** [comment]
I’m starting to wonder whether this might be a timing issue. Given the code involved, the only situation where this should be possible is if the build workflow is getting triggered on the correct commit, but the associated tag is _not_ present in the copy of the repo that it’s checking out.

We’re definitely pushing the tag correctly (see the output of the ‘Commit Changes’ step in https://github.com/netdata/netdata/runs/6829998242?check_suite_focus=true, it clearly shows the tag being successfully pushed, and on top of that the tag _is_ there in the repo), which suggests to me that something internal to GitHub is using an eventual consistency model for this type of thing, and we’re happening to check out the repo before everything is actually consistent.

This would also explain why we’re having naming issues for the build artifacts and having to manually fix them, though it begs the question of why it0s only affecting the build workflow, and not the Docker or native package builds.

**Comment by Ferroin** [comment]
I’ve still not figured out a root cause, but have at least figured out a workaround on our end to hopefully prevent this from happening again.

I’m leaving this open until we have a patch release published that actually has the correct version number.

**Comment by tms0** [comment]
Hi @Ferroin,

It seems fixed on v1.36.1.
The displayed version is correct and the kickstarter script is working as expected.

Thank you :-)

**Comment by MrZammler** [comment]
Hey @tms0 thank you very much for the follow up!
